package-lock.json

```diff
@@ -1,8 +1,10288 @@
 {
   "name": "aria-practices",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Upgrading the lockfile might make sense, but probably better to do it separately

Reply:
https://github.com/w3c/aria-practices/pull/2108

Reply:
That is now merged, so this needs a rebase. cc @howard-e

Reply:
Done